nixos/modules/services/databases/arangodb.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.arangodb;
+
+in {
+
+  ###### interface
+
+  options = {
+    services.arangodb = {
+      enable = mkEnableOption "ArangoDB server";
+
+      dataDir = mkOption {
+        type = types.path;
+        example = "/var/db/arangodb";
+        default = "/var/db/arangodb";
+        description = "ArangoDB data dir";
+      };
+    };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+    users.extraUsers.arangodb = {
+      name = "arangodb";
+      uid = config.ids.uids.arangodb;
+      group = "arangodb";
+      description = "ArangoDB server user";
+    };
+
+    users.extraGroups.arangodb.gid = config.ids.gids.arangodb;
+
+    environment.systemPackages = [ pkgs.arangodb ];
```

Comment:
You shouldn't need to add the `arangodb` package system wide. See comment below...

Reply:
Okay, I have removed it

Reply:
But if I remove it, I don't have access to arangosh and else